frontend/src/reducers/plannerSlice.js

```diff
@@ -85,13 +85,14 @@ const plannerSlice = createSlice({
       Object.keys(action.payload).forEach((course) => {
         if (state.courses[course]) {
           const {
-            is_accurate: isAccurate, unlocked, warnings, handbook_note: handbookNote,
+            is_accurate: isAccurate, unlocked, warnings, handbook_note: handbookNote, supressed,
           } = action.payload[course];
 
           state.courses[course].isAccurate = isAccurate;
           state.courses[course].isUnlocked = unlocked;
           state.courses[course].warnings = warnings;
           state.courses[course].handbookNote = handbookNote;
+          state.courses[course].supressed = supressed;
```

Comment:
Since you're modifying the planner state structure, might want to increment the data version in `constants.js` for a clean refresh.

Reply:
hmm - i dont think it matters because we immediately call `validateTermPlanner` which will fill in this data on the fly. So no need to increment,